contracts/cw-proposal-multiple/check_wasm.sh

```diff
@@ -0,0 +1,5 @@
+#!bin/bash
+
+RUSTFLAGS='-C link-arg=-s' cargo wasm
+cp ../../target/wasm32-unknown-unknown/release/cw3_multiple_choice.wasm .
+check_contract cw3_multiple_choice.wasm
```

Comment:
Where is this `check_contract` function defined?

Reply:
sorry, this was for testing